doc/Analytics.xml

```diff
@@ -3654,6 +3518,39 @@ Added Recognition rule engines</revremark>
     <section>
       <title>Object Detection</title>
       <para>The object detection rule generates events when any of the configured object types are detected in the field of view. Whenever there is change in the detected object types, a new event is notified, if none of the objects from the configured list is detected, an event with empty object list is notified.</para>
+      <variablelist role="op">
+        <varlistentry>
+          <term>Parameters</term>
+          <listitem>
+            <para role="param">ClassFilter [tt:StringList]</para>
+            <para role="text">List of classes to be detected.</para>
+            <para role="param">ConfidenceLevel - optional [xs:float]</para>
+            <para role="text">Minimum confidence level for accepting a classification. </para>
+            <para role="param">DwellTime - optional [xs:duration]</para>
+            <para role="text">Duration an object must be visible before accepting a classification.</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>Topic</term>
+          <listitem>
+            <para role="param">tns1:RuleEngine/LoiteringDetector/ObjectDetection/Object</para>
```

Comment:
Topic in this location needs correction, in example xml its correct.

tns1:RuleEngine/LoiteringDetector/ObjectDetection/Object  
to 
tns1:RuleEngine/ObjectDetection/Object